app/views/navbar.scala.html

```diff
@@ -29,10 +29,26 @@
                     <li>
                         <a href="#" id="toolbar-onboarding-link">Retake tutorial</a>
                     </li>
-                }
-
-
 
+                }
+                @if(url.get == "/audit"){
+                    <script type="text/javascript">
```

Comment:
Could you possibly put this in a css file? Similar to .navbar-xs. And add that class to the buttons? It would be cleaner than manipulating it within the code.